packages/electron/app/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "resolutions": {
     "electron": "3.1.0",
-    "debug": "^2.6.9"
+    "debug": "^2.6.9",
+    "xpc-connection": "sandeepmistry/node-xpc-connection#26/head"
```

Comment:
I think that committing this will break the build for windows. This is added in a post-install script on MacOS.